### DISCUSSION TITLE
Thank you for this

### DISCUSSION THREAD

**Discussion by Blackchariotnetwork** [AAndyProgram/SCrawler#22] [General]
Just wanted to thank you for making this! Saved me a ton of time and easy to set up.

**Comment by AAndyProgram** [comment]
Thank you for those kind words! Glad you like it!

**Reply by nerzak-dominion** [reply]
Same as OP. Thank you for this, great tool!

**Reply by AAndyProgram** [reply]
Thank you! 😊
